bayesian/utils.py

```diff
@@ -64,17 +64,19 @@ def get_user_email(user_profile):
 
 
 def create_component_bookkeeping(ecosystem, packages_list,
-                                 user_id, user_agent, manifest_hash, request_id):
+                                 source, manifest_hash, headers):
     """Run the component analysis for given ecosystem+package+version."""
     args = {
-        'external_request_id': request_id,
+        'external_request_id': headers.get('X-Request_Id', None),
```

Comment:
X-Request_Id is incorrect.
```suggestion
        'external_request_id': headers.get('X-Request-Id', None),
```

Reply:
Done.